makefile

```diff
@@ -115,7 +115,7 @@ dexrt-llvm: src/lib/dexrt.bc
 
 # --- running tests ---
 
-example-names = mandelbrot pi sierpinski rejection-sampler \
+example-names = simplex mandelbrot pi sierpinski rejection-sampler \
```

Comment:
Could you please move this example to the end? The ones at the front are usually quite simple and usually much better for debugging.

Reply:
Done.